code-challenges.test.js

```diff
@@ -49,12 +82,29 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
+// describe("titleCase", () => {
+//     it("Create a function that takes in an array of words and returns an array with all the words capitalized.", () => {
+//         const randomNouns4 = ["streetlamp"]
+//         expect(titleCase(randomNouns4)).toEqual('Streetlamp')
+//     })
+// })
+
 // b) Create the function that makes the test pass.
+const titleCase = (Array) => {
+    return Array.map( value => value.replace(value[0],value[0].toUpperCase()))
+}
+
+console.log(titleCase(randomNouns1));
+console.log(titleCase(randomNouns2));
 
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // Pseudo code:
 
+// input: strings
+// output : the index of the first vowel in the string
+// process: Will use regex and .search. Since that returns the index of the match
```

Comment:
This pseudo code is better than the last two challenges! I would elaborate some more though. Along with what you're using, how does it work?